acceptance/acceptance/mocks/CheckInformationMock.scala

```diff
@@ -0,0 +1,28 @@
+package acceptance.mocks
+
+import model.{CheckInformation, ContactDetails, TermsOfUseAgreement}
+import org.joda.time.DateTime
+
+trait CheckInformationMock {
```

Comment:
Can we call this something like Fake as Mock implies something else..

Reply:
We are changing them to test data